### PR TITLE
Add alertmanager resources

### PR DESCRIPTION
**Pull request by ArthurSens**
## Related Issue(s)

Fixes #

## How to test

```console
go run installer/main.go render > output.yaml
code output.yaml
```

Verity prometheus manifests are being rendered as expected

### CODE REVIEW

File: installer/pkg/components/alertmanager/constants.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package alertmanager
+
+const (
+	Name      = "main"
```
Comment (mads-hartmann) [inline installer/pkg/components/alertmanager/constants.go:6]: Same question as https://github.com/gitpod-io/observability/pull/203#discussion_r941043351 ☺️

File: installer/pkg/components/alertmanager/role.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+				Name:      Name,
+				Namespace: Namespace,
+				Labels:    common.Labels(Name, Component, App, Version),
+			},
+			Rules: []rbacv1.PolicyRule{
+				{
+					APIGroups: []string{"policy"},
+					Resources: []string{"podsecuritypolicies"},
+					Verbs:     []string{"use"},
+					// TODO: The psp name will be a constant declared in the 'common' pkg.
```
Comment (mads-hartmann) [inline installer/pkg/components/alertmanager/role.go]: Same question as https://github.com/gitpod-io/observability/pull/201#discussion_r941033700

**Review by mads-hartmann — COMMENTED** [review]
Two minor comments I'd like to clarify before I can approve ☺️

File: installer/pkg/components/alertmanager/constants.go (hunk 1 above)
Comment (ArthurSens) [reply installer/pkg/components/alertmanager/constants.go:6]: That's the name that comes from upstream, I'm just copy-pasting 😅. The Alertmanager CR will be named `main`, then the statefulset created by the operator will be named `alertmanager-main`.

You do have a good point that all the resources that are not created by the operator will get awkward name... let me fix that

File: installer/pkg/components/alertmanager/role.go (hunk 2 above)
Comment (ArthurSens) [reply installer/pkg/components/alertmanager/role.go]: done!

**Review by meysholdt — APPROVED** [review]
Code changes LGTM

**Pull request merged**